docs/commands/exit.md

```diff
@@ -0,0 +1,17 @@
+# exit
+
+Exits the nu shell. If STATUS is supplied, it will be converted to an integer and used as the exit code. Otherwise, the exit code will be that of the last command executed.
+
+## Examples 
+
+```shell
+> exit
+```
+
+```shell
+> exit 1
```

Comment:
This is how exit works in bash, but not how exit works in Nu.

In Nu, exit works either:
* `exit` <-- exits the current shell
* `exit --now` <-- exits out of Nu

Reply:
done